packages/jbr/package.json

```diff
@@ -66,6 +66,7 @@
     "@types/cross-spawn": "^6.0.2",
     "@types/dockerode": "^3.2.3",
     "@types/fs-extra": "^9.0.11",
+    "@types/semver": "^7.5.1",
```

Comment:
Could you also add `semver` as dependency? Weird that the linter did not complain about this.

Reply:
Good point, added it. Yeah, usually I just wait for the linter to complain about dependencies and add them.